test_aeo/dsgrid_project/datasets/benchmark/Commercial_End_Use_Growth_Factors/dataset.toml

```diff
@@ -1,16 +1,16 @@
 data_source = "AEO2021 Reference Case"
 dataset_id = "aeo2021_ref_com_energy_end_use_growth_factors"
-data_schema_type = "one_table"
-dataset_type = "benchmark" # <--- ?
-path = "/Users/lliu2/Documents/GitHub/dsgrid-project-StandardScenarios/dsgrid_project/datasets/benchmark/aeo2021/reference/commercial/End Use Growth Factors/data/" # <---
+data_schema_type = "growth_rate"
+dataset_type = "benchmark"
+path = "/Users/lliu2/Documents/GitHub/dsgrid-project-StandardScenarios/dsgrid_project/datasets/benchmark/aeo2021/reference/commercial/End Use Growth Factors/data/"
 description = """
 AEO 2021 Reference Case Commercial Energy End Use Annual Growth Factors. 
 Detailed data provided by EIA, not available in API
 """
 origin_creator = "Lixi Liu"
 origin_organization = "NREL"
 origin_contributors = [ "Lixi Liu", "Meghan Mooney", "Elaine Hale",]
-origin_project = "Energy Benchmarking and Projection" # <---
+origin_project = "Energy Benchmarking and Projection"
```

Comment:
This should be something specific. How about "dsgrid 2022 commercial buildings projection"?